mmv1/products/spanner/Database.yaml

```diff
@@ -153,10 +153,16 @@ properties:
       - !ruby/object:Api::Type::String
         name: 'kmsKeyName'
         immutable: true
-        required: true
         description: |
           Fully qualified name of the KMS key to use to encrypt this database. This key must exist
           in the same location as the Spanner Database.
+      - !ruby/object:Api::Type::Array
```

Comment:
Should we add https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#exactly_one_of for these 2 fields?